libs/frontend/util-edit-text/src/lib/schema-basic.ts

```diff
@@ -33,11 +33,11 @@ export const nodes = {
   } as NodeSpec,
 
   /// A plain paragraph textblock. Represented in the DOM
-  /// as a `<p>` element.
+  /// as a `<div>` element so that i18n nodes can be nested
```

Comment:
I changed the wrapper component for text from `p` to `div`. The main reason is that since `i18n` entries are also edited with ProseMirror, including a translation in a component  would result in nested `<p>`s, which is not allowed.

I'm still not that happy with how I implemented the `i18n` feature, but it's getting better.